test-suite/output/notation_existing_level.out

```diff
@@ -0,0 +1,16 @@
+File "./output/notation_existing_level.v", line 3, characters 0-80:
+The command has indeed failed with message:
+Ignoring "at existing level", no previously reserved notation with a common prefix.
```

Comment:
"Ignoring" sounds like a warning, not an error (not fatal).

Reply:
Yes it indeed is a warning. It's just made an error for the test (c.f. the `#[warning="+..."]` in the .v file).